src/dashboard/events/cd-dashboard-upcoming-event.vue

```diff
@@ -2,8 +2,9 @@
   <div class="cd-dashboard-upcoming-event">
     <div v-if="isLoaded" class="cd-dashboard-upcoming-event__content" :class="{ 'cd-dashboard-upcoming-event__content--booked': hasOrder}">
       <div class="cd-dashboard-upcoming-event__main">
-        <h4 v-if="hasOrder">{{ $t('Your next event is "{name}"', { name: event.name }) }}</h4>
-        <h4 v-else>{{ $t('"{name}" is the next session available', { name: event.name }) }}</h4>
+        <h4 v-if="hasOrder">{{ $t('"{name}" is booked', { name: event.name }) }}</h4>
+        <h4 v-else-if="remainingTickets > 0">{{ $t('"{name}" is fully booked', { name: event.name }) }}</h4>
+        <h4 v-else>{{ $t('"{name}" is the available to book', { name: event.name }) }}</h4>
```

Comment:
Drop the "the", it's cleaner.

Also, it makes it grammatically correct. 😜

Reply:
Duh oO' My bad